test/find-and-lock-next-job.js

```diff
@@ -24,42 +22,32 @@ const jobType = "do work";
 const jobProcessor = () => {};
 
 describe("find-and-lock-next-job", () => {
-  beforeEach((done) => {
+  beforeEach(async () => {
+    mongoCfg = await getMongoCfg(agendaDatabase);
+  });
+
+  beforeEach(async () => {
     agenda = new Agenda(
       {
         db: {
           address: mongoCfg,
         },
-      },
-      (error) => {
-        if (error) {
-          done(error);
-        }
-
-        MongoClient.connect(
-          mongoCfg,
-          { useUnifiedTopology: true },
-          async (error, client) => {
-            mongoClient = client;
-            mongoDb = client.db(agendaDatabase);
-
-            await delay(50);
-            await clearJobs();
-
-            agenda.define("someJob", jobProcessor);
-            agenda.define("send email", jobProcessor);
-            agenda.define("some job", jobProcessor);
-            agenda.define(jobType, jobProcessor);
-
-            done();
-          }
-        );
-      }
-    );
+      });
+  
+      await jobs._ready;
```

Comment:
There is no jobs here. Did you mean agenda? Related to https://github.com/agenda/agenda/issues/1369.

Reply:
That's an after merge bug. Could you please fix it in `master` @harisvsulaiman ?